telemetry/specs/INT.mdk

```diff
@@ -1330,6 +1483,31 @@ INT Metadata Header and Metadata Stack, followed by TCP payload:
    |                          TCP payload                          |
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
```

Comment:
IMO it would be cleaner to add a section header. Change the title of the previous example to "Example with INT-MD over TCP", then add a title here for "Example with INT-MX over TCP", referring to the previous section for the IP and TCP headers. Start with the INT shim header with type = 3 and length = 3 here.

Reply:
Done.